package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "synapse-react-client",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

```suggestion
  "version": "2.1.0",
```

I didn't release 2.1.0, so this is unnecessary (unless you did)